ui/app/mirrors/create/cdc/schemabox.tsx

```diff
@@ -199,12 +199,17 @@ export default function SchemaBox({
               ?.map((tableMap) => tableMap.sourceTableIdentifier)
               .includes(row.source)
           ) {
+            const existingRow = alreadySelectedTables?.find(
```

Comment:
```suggestion
            const existingRow = alreadySelectedTables.find(
```
can remove the `?.map` above too. As is it'll hit an error about trying to call `undefined` as a function. You'd have to use the `find?.(...)` to null check the call too